lib/_stream_readable.js

```diff
@@ -45,7 +45,7 @@ var Stream = require('./internal/streams/stream');
 
 var Buffer = require('buffer').Buffer;
 
-var OurUint8Array = global.Uint8Array || function () {};
+var OurUint8Array = Uint8Array || function () {};
```

Comment:
this dose not quite work... you can't do `var x = foobar || function() {}`
you need to do: `var x = typeof foobar !== 'undefined' ? foobar : function() {}`

Checking if something exist globally must be done either via a global object like window, self, globalThis etc or using `typeof` checks like i suggested

Reply:
Got it, thanks for the explanation.